ios/Podfile

```diff
@@ -10,12 +10,14 @@ workspace 'jitsi-meet'
 
 install! 'cocoapods', :deterministic_uuids => false
 
+use_modular_headers!
```

Comment:
Do we need this?

Reply:
Yes, we need it for tackling dependencies that Firebase depends on.

https://stackoverflow.com/questions/72289521/swift-pods-cannot-yet-be-integrated-as-static-libraries-firebasecoreinternal-lib